docs/videos/running_all_devs.md

```diff
@@ -0,0 +1,41 @@
+# Running the All Core Devs call
+
+Create draft agenda file titled “alldevs[date].md” with the date of the next call in YYYYMMDD format.
+- Update the mkdocs.yml file of the wiki with a relative link to the file under the Videos > Meetings > All Core Devs > Agendas section.   
+- Add link to the agenda to the [Overview](dev.md) page in the wiki.
+
+Review proposed agenda items
+ - Check alldevs email address throughout the week for proposed agenda items. Review proposed items and respond with [accepted](all_devs_accepted.md) or [rejected](all_devs_rejected.md) templates.
+ - Add approved agenda items and the email addresses of agenda item owners to this meeting’s “Agenda + Guest Participants for [date]” email, replacing [date] with the date of the meeting.
+  - All approved agenda items to the next call’s agenda in the wiki.
+
+Ten mins before scheduled to begin. Create [Jitsi](https://meet.jit.si) channel with unique link.
+
+Five mins before scheduled to begin. Send email with all participants bcc’d.
```

Comment:
Could we just use a calendar invite instead of an email?

Right now I update the calendar item for the call with the agenda and add any new participants that way.

Reply:
That's fine if we're ok with everyone on the call knowing everyone else's email addresses.

Reply:
I think that's ok, especially as it'd make it easier for people who want to contact each other afterwards due to some meeting points / etc. Worst case they can use a different email address for this invite if they'd like.